libs/model/test/contest/check-contests.spec.ts

```diff
@@ -168,7 +168,7 @@ describe('Check Contests', () => {
     );
 
     await handleEvent(ContestWorker(), {
-      name: EventNames.CheckContests,
+      name: EventNames.ContestRolloverTimerTicked,
```

Comment:
do you need this twice?

Reply:
in the test, yes

it verifies how the check behaves based on the contest state